spec/spec_helper.rb

```diff
@@ -5,6 +5,9 @@
 
 ENV["RAILS_ENV"] = "test"
 
+require "dotenv"
+Dotenv.overload(".sample.env")
```

Comment:
Ah, that actually makes sense. The `Dotenv.overload` is before the environment is loaded so that the proper environment variables will be used in the initializers. This also means that the `Bundler.require` line hasn't loaded the gems yet.